### PR TITLE
Nodejs Week1 

### PR DESCRIPTION
**Pull request by bengu-thon-mai-mochi**
Use modules with Express and serve JSON data.

### CODE REVIEW

File: nodejs/week1/meal_sharing/src/backend/index.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+const server = express();
+
+const cheapMeals = require('./routes/cheap-meals');
+const largeMeals = require('./routes/large-meals');
+const meals = require('./routes/meals');
+const meal = require('./routes/meal');
+const reservation = require('./routes/reservation');
+const reservations = require('./routes/reservations');
+
+server.set('port', process.env.PORT || 2000);
```
Comment (Hatko) [inline nodejs/week1/meal_sharing/src/backend/index.js]: `process.env.PORT` 👍

File: nodejs/week1/meal_sharing/src/backend/routes/cheap-meals.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
+const express = require('express');
+const router = express();
+
+const mealsJson = require(__dirname + '/../data/meals.json');
+const reviewsJson = require(__dirname + '/../data/reviews.json');
```
Comment (Hatko) [inline nodejs/week1/meal_sharing/src/backend/routes/cheap-meals.js]: It seems unused

File: nodejs/week1/meal_sharing/src/backend/routes/large-meals.js [hunk 3]
```diff
@@ -0,0 +1,13 @@
+const express = require('express');
+const router = express();
+
+const mealsJson = require(__dirname + '/../data/meals.json');
+const reviewsJson = require(__dirname + '/../data/reviews.json');
```
Comment (Hatko) [inline nodejs/week1/meal_sharing/src/backend/routes/large-meals.js]: It seems unused

File: nodejs/week1/meal_sharing/src/backend/routes/meals.js [hunk 4]
```diff
@@ -0,0 +1,20 @@
+const express = require('express');
+const router = express();
+
+const mealsJson = require(__dirname + '/../data/meals.json');
+const reviewsJson = require(__dirname + '/../data/reviews.json');
+
+mealsJson.forEach((meal) => {
+   meal.reviews = [];
```
Comment (Hatko) [inline nodejs/week1/meal_sharing/src/backend/routes/meals.js]: Please check this comment:

https://github.com/krithi-kondana/hyf-homework/pull/30#discussion_r382179162

Comment (bengu-thon-mai-mochi) [reply nodejs/week1/meal_sharing/src/backend/routes/meals.js]: Thank you I was actually working on it today. I will integrate the changes you mentioned. I was wondering if there is a possibility that we can extract the functions with similar logics to into routers? For example throwing a random meal/app share the same function, can we use them as modules too?

**Pull request merged**